types/chrome/index.d.ts

```diff
@@ -5987,7 +6000,7 @@ declare namespace chrome.printerProvider {
 
     export interface PrinterCapabilities {
         /** Device capabilities in CDD format. */
-        capabilities: any;
+        capabilities: object;
```

Comment:
It's an object representing printer capabilities in CDD format.

Reply:
`object` and its sister type, `Object`, are very ambiguous types, as pointed out [here](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/ban-types.md#default-options). I'm not fully sure what you're going for here, but here's the error message that typescript-eslint will give for some guidance:
```
The `Object` type actually means "any non-nullish value", so it is marginally better than `unknown`.
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
```